server/map_rpc_server.go

```diff
@@ -57,13 +57,13 @@ func (t *TrillianMapServer) GetLeaves(ctx context.Context, req *trillian.GetMapL
 
 	tree, hasher, err := t.getTreeAndHasher(ctx, mapID, true /* readonly */)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("getTreeAndHasher(): %v", err)
```

Comment:
Could you use textual descriptions instead of the function name? Eg, `"error creating tree hasher: %v"`. I expect those to be more human-friendly, especially for people not familiar with Trillian inner workings.

Ditto for all other changed messages.